model/src/main/java/com/basistech/rosette/dm/AttributeKey.java

```diff
@@ -24,13 +24,13 @@
  */
 enum AttributeKey {
     BASE_NOUN_PHRASE("baseNounPhrases"),
+    CATEGORIZER_RESULTS("categorizerResults"),
+    DEPENDENCY("dependency"),
+    EMBEDDING("embeddings"),
```

Comment:
I did this on purpose by hand to make the list more maintainable. It looked to me as if some people on some days has sorted some parts.